SwiftCompilerSources/Sources/Optimizer/FunctionPasses/ReleaseDevirtualizer.swift

```diff
@@ -44,9 +46,11 @@ let releaseDevirtualizerPass = FunctionPass(
           // _not_ released by the deinit method.
           if let deallocStackRef = instruction as? DeallocStackRefInst {
             if !context.continueWithNextSubpassRun(for: release) {
-              return
+              return // Question: should this be continue?
```

Comment:
No, `return` is the right thing to do. It should abort the pass.

Reply:
thank you a lot Eric